src/utils/DOMUtils.js

```diff
@@ -15,12 +15,16 @@ export const clearDOMElement = (DOMElement) => {
  */
 export const createDOMElement = (tag, options) => {
   const { id } = options || {};
+  const { className } = options || {};
```

Comment:
```suggestion
export const createDOMElement = (tag, options = {}) => {
  const { id, className } = options;
```
Setting `options = {}` as a parameter means that it will default to `{}` if `options` is undefined